.github/workflows/test.yml

```diff
@@ -0,0 +1,41 @@
+name: VetClinicPublic CI Pipeline
```

Comment:
What is this pipeline ?

Reply:
this pipeline came from my base branch with the automation, and it got pushed to my feature/ci-cd